tests/__init__.py

```diff
@@ -1 +1,14 @@
 from __future__ import absolute_import, division, print_function
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+# create log handler for tests
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(levelname)s - %(message)s')
```

Comment:
I'm fond of `"%(levelname).1s %(name)s %(message)s"` if it's all the same to you.

Reply:
Done.

Reply:
Btw..this is for the root logger for the test. The test output will look as follows (I added a newline before message). Seems a little cluttered to me. What do you think?


```
I tests.nn.test_clipped_nn 
 softmax_ps = Variable containing:
 0.2689  0.7311
[torch.FloatTensor of size 1x2]

.I tests.nn.test_clipped_nn 
 epsilon = 1e-05
I tests.nn.test_clipped_nn 
 softmax_ps = Variable containing:
 0.2689  0.7311
[torch.DoubleTensor of size 1x2]
```

Reply:
It would be cleaner to log `softmax_ps.data.cpu().numpy()`, but that seems to be an issue for the user rather than the log handler.

Reply:
That was just a random sample that I picked. Changing this to a tab instead of a newline.